modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- The `name` and `id` attributes of any `img` or `iframe` elements were incorrectly removed #TINY-8773
```

Comment:
```suggestion
- The `name` and `id` attributes of some elements were incorrectly removed during serialization #TINY-8773
```
This can happen with other elements, so this just makes it more generic.